plenum/test/monitoring/test_monitoring_params_with_zfn.py

```diff
@@ -4,8 +4,20 @@
 
 nodeCount = 7
 
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    old_thr_window_size = tconf.ThroughputInnerWindowSize
+    old_thr_window_count = tconf.ThroughputThresholdWindowCount
+    tconf.ThroughputInnerWindowSize = 5
+    tconf.ThroughputThresholdWindowCount = 2
+
+    yield tconf
+    tconf.ThroughputInnerWindowSize = old_thr_window_size
+    tconf.ThroughputThresholdWindowCount = old_thr_window_count
+
 
 def testThroughputThreshold(looper, txnPoolNodeSet, requests):
+    looper.runFor(10)
```

Comment:
Maybe use `tconf.ThroughputInnerWindowSize *tconf.ThroughputThresholdWindowCount` instead of 10?